storage/spec/resources/enc_file.txt

```diff
@@ -0,0 +1 @@
+Content of test enc_file.txt
```

Comment:
To keep things simpler, how about using the existing file.txt?

As far as I can tell, there is no reason for the encryption tests to upload/download a different file than the non-encrypted samples.